src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/FundingComplete.cshtml

```diff
@@ -1,22 +1,14 @@
 ﻿@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
 
-@if (Model.ReservedFundingToShow != null)
-{
+
     <h3 class="das-panel__heading">Apprenticeship funding secured</h3>
     <dl class="das-definition-list das-definition-list--with-separator">
         <dt class="das-definition-list__title">Legal entity:</dt>
-        <dd class="das-definition-list__definition">@Model.ReservedFundingToShowLegalEntityName</dd>
+        <dd class="das-definition-list__definition">@(Model.AccountViewModel.Name)</dd>
```

Comment:
This can display the incorrect legal entity name for the reservation being shown (see other comment for more details).

Reply:
Yup, Should've done some mumbo jumbo to make it work... hopefully